### PR TITLE
replaced the EmptyState image with a pure css version + improved styling of the texts

### PR DESCRIPTION
**Pull request by pegahvaezi**

### CODE REVIEW

File: src/components/EmptyState/EmptyState.css [hunk 1]
```diff
@@ -1,34 +1,339 @@
...
+  z-index: 99;
+}
+a {
+  color: #d67b32;
+  text-decoration: none;
+  cursor: pointer;
+}
+a:hover {
+  text-decoration: underline;
+}
```
Comment (Connoropolous) [inline src/components/EmptyState/EmptyState.css]: I don't think we want a globally scoped rule that ill apply to all links coming in.... what is this for, and where does it come from?

File: src/components/EmptyState/EmptyState.css [hunk 2]
```diff
@@ -1,34 +1,339 @@
...
+  font-weight: bold;
+  display: block;
+  letter-spacing: 2px;
+  max-width: 100%;
+  text-align: center;
+  position: absolute;
+  bottom: 20px;
+  left: 30px;
+  z-index: 99;
+}
```
Comment (Connoropolous) [inline src/components/EmptyState/EmptyState.css]: this doesn't seem to apply to any code

File: src/components/EmptyState/EmptyState.js [hunk 3]
```diff
@@ -2,16 +2,49 @@ import './EmptyState.css'
...
+  /* <img src='img/squirrel_illustration_empty.png' /> */
   return (
     <div className='empty_state_wrapper'>
       <div className='empty_state'>
-        <img src='img/squirrel_illustration_empty.png' />
-        <h1>welcome to acorn</h1>
-        <h4>
+        <div class='outer'>
+          <div class='middle'>
+            <div class='main clearfix'>
```
Comment (Connoropolous) [inline src/components/EmptyState/EmptyState.js]: can you rename the 'main' class here and in the css. It's too generic. It doesn't matter what you call it, just make it something less universal, and more specific

**Review by Connoropolous — CHANGES_REQUESTED** [review]

File: src/components/EmptyState/EmptyState.css (hunk 1 above)
Comment (pegahvaezi) [reply src/components/EmptyState/EmptyState.css]: I think that's a mistaken added line. we can remove it.

File: src/components/EmptyState/EmptyState.js (hunk 3 above)
Comment (pegahvaezi) [reply src/components/EmptyState/EmptyState.js]: ok

File: src/components/EmptyState/EmptyState.css (hunk 2 above)
Comment (pegahvaezi) [reply src/components/EmptyState/EmptyState.css]: removed

File: src/components/EmptyState/EmptyState.css [hunk 4]
```diff
@@ -1,34 +1,347 @@
...
+
+.clearfix::after {
+  content: '';
+  visibility: hidden;
+  display: inline-block;
+  font-size: 0;
+  clear: both;
+  height: 0;
+}
```
Comment (Connoropolous) [inline src/components/EmptyState/EmptyState.css]: one more thing... this 'clearfix' is a commonly used practice across web development for using a class name to handle situations where they are using float properties for css positioning. It's kind of like a global rule too, so maybe if we are going to keep it you could move it into `index.css` or `App.css`

Comment (pegahvaezi) [reply src/components/EmptyState/EmptyState.css]: removed it-- we actually didnt need it

File: src/components/EmptyState/EmptyState.css [hunk 5]
```diff
@@ -1,34 +1,334 @@
...
+    transform: scale(0.75);
   }
+}
 
-   .empty_state h4 {
-    font-family: 'quebecregular', Helvetica, Sans-Serif;
-    font-weight: normal;
-    margin: 5px auto;
+@media (max-width: 420px) {
+  .main {
```
Comment (Connoropolous) [inline src/components/EmptyState/EmptyState.css]: you missed this one

**Review by Connoropolous — CHANGES_REQUESTED** [review]

File: src/components/EmptyState/EmptyState.css (hunk 5 above)
Comment (pegahvaezi) [reply src/components/EmptyState/EmptyState.css]: done

**Review by Connoropolous — APPROVED** [review]

**Pull request merged**